verify/src/main/java/com/twilio/verify/domain/factor/PushFactory.kt

```diff
@@ -163,6 +163,13 @@ internal class PushFactory(
     }
   }
 
+  fun deleteAllFactors(then: () -> Unit) {
```

Comment:
Is there a way to delete all factors at once instead of one by one?

Reply:
As we need to delete the key pairs for the factors, we need a for to do this, we can call the `FactorProvider.clear`